.env.development

```diff
@@ -49,3 +49,5 @@ EDS_CACHE_DIR=tmp/cache
 
 EBSCO_SEARCH_URL="define EBSCO_SEARCH_URL"
 FINDING_AIDS_SEARCH_URL="define FINDING_AIDS_SEARCH_URL"
+
+GLOBAL_MESSAGE_URL="define GLOBAL_MESSAGE_ENDPOINT"
```

Comment:
should this be "define GLOBAL_MESSAGE_URL" since it looks like ENDPOINT is not defined?

Reply:
yeah, changed the variable name, but forgot to update the placeholder. good catch!